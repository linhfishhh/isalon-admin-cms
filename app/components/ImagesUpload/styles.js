import styled from 'styled-components';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles(theme => ({
  root: {
    display: 'flex',
    flexWrap: 'wrap',
    justifyContent: 'space-around',
    overflow: 'hidden',
    backgroundColor: theme.palette.background.paper,
  },
  gridList: {
    flexWrap: 'nowrap',
    transform: 'translateZ(0)',
    width: '100%',
  },
  title: {
    color: 'white',
    height: '100%',
  },
  titleBar: {
    background: 'transparent',
  },
  icon: {
    color: 'white',
    width: 60,
    height: 60,
  },
}));

const InputFile = styled.input`
  width: 0.1px;
  height: 0.1px;
  opacity: 0;
  overflow: hidden;
  position: absolute;
  z-index: -1;
`;
const Label = styled.label`
  max-width: 100%;
  font-size: ${props => props.theme.typography.body1.fontSize};
  text-overflow: ellipsis;
  white-space: nowrap;
  cursor: pointer;
  display: inline-block;
  overflow: hidden;
  padding: 0.625rem 1.25rem;
  text-align: center;
  svg {
    vertical-align: middle;
    fill: currentColor;
    margin-top: -0.25em;
    margin-right: 0.25em;
  }
  figure {
    width: 100px;
    height: 100px;
    border-radius: 50%;
    background-color: ${props => props.theme.palette.primary.main};
    display: block;
    padding: 20px;
    margin: 0 auto 10px;
  }
  span {
    min-height: 2em;
    display: inline-block;
    text-overflow: ellipsis;
    white-space: nowrap;
    overflow: hidden;
    vertical-align: top;
  }
`;

const DraggableWrapper = styled.div`
  display: flex;
  overflow: auto;
  max-width: ${props => props.width}px;
  width: ${props => props.width}px;
`;

const DraggableItem = styled.div`
  user-select: none;
  padding: 2px;
  min-width: 150px;
  width: ${props => props.width}px;
  height: ${props => props.height}px;
`;

const ImageStyle = styled.img`
  width: 100%;
  height: 100%;
  object-fit: cover;
  border: solid 1px #ddd;
`;

export {
  useStyles,
  InputFile,
  Label,
  DraggableWrapper,
  DraggableItem,
  ImageStyle,
};
