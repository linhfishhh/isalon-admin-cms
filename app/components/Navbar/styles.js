import styled from 'styled-components';
import { darken } from 'polished';
import { InputBase, AppBar as MuiAppBar } from '@material-ui/core';

const AppBar = styled(MuiAppBar)`
  background: ${props => props.theme.header.background};
  color: ${props => props.theme.header.color};
`;

const Search = styled.div`
  border-radius: 2px;
  background-color: ${props => props.theme.header.background};
  display: none;
  position: relative;
  width: 100%;

  &:hover {
    background-color: ${props => darken(0.05, props.theme.header.background)};
  }
`;

const SearchIconWrapper = styled.div`
  width: 50px;
  height: 100%;
  position: absolute;
  pointer-events: none;
  display: flex;
  align-items: center;
  justify-content: center;

  svg {
    width: 22px;
    height: 22px;
  }
`;

const Input = styled(InputBase)`
  color: inherit;
  width: 100%;

  > input {
    color: ${props => props.theme.header.search.color};
    padding-top: ${props => props.theme.spacing(2.5)}px;
    padding-right: ${props => props.theme.spacing(2.5)}px;
    padding-bottom: ${props => props.theme.spacing(2.5)}px;
    padding-left: ${props => props.theme.spacing(12)}px;
    width: 160px;
  }
`;

export { SearchIconWrapper, Input, Search, AppBar };
