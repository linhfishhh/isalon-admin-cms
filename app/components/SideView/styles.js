import styled from 'styled-components';

const OverlaySideView = styled.div`
  display: ${props => (props.show ? 'block' : 'none')};
  width: 100%;
  height: 100%;
  top: 0;
  left: 0;
  position: absolute;
  z-index: 2;
  background-color: rgba(0, 0, 0, 0.3);
`;

const WrapperSideView = styled.div`
  height: ${props => props.size}px;
  width: 100%;
  position: absolute;
  z-index: 10;
  bottom: 0;
  left: 0;
  background-color: #fff;
  border-top: solid 1px #ccc;
  box-shadow: 0 0 4px 0 #ccc;
  overflow-x: hidden;
  transition: 0.5s;
`;

const ButtonClose = styled.div`
  position: absolute;
  top: 5px;
  right: 5px;
`;

export { OverlaySideView, WrapperSideView, ButtonClose };
