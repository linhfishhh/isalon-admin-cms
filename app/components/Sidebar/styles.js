import styled from 'styled-components';
import {
  ListItem,
  Typography,
  Drawer as MuiDrawer,
  ListItemText,
  List as MuiList,
} from '@material-ui/core';
import { ExpandLess, ExpandMore } from '@material-ui/icons';
import { darken, rgba } from 'polished';
import PerfectScrollbar from 'react-perfect-scrollbar';

const Drawer = styled(MuiDrawer)`
  border-right: 0;
  > div {
    border-right: 0;
  }
`;
const Brand = styled(ListItem)`
  font-size: ${props => props.theme.typography.h5.fontSize};
  font-weight: ${props => props.theme.typography.fontWeightMedium};
  color: ${props => props.theme.sidebar.header.color};
  background-color: ${props => props.theme.sidebar.header.background};
  font-family: ${props => props.theme.typography.fontFamily};
  min-height: 56px;

  ${props => props.theme.breakpoints.up('sm')} {
    min-height: 64px;
  }
`;

const SidebarFooter = styled.div`
  background-color: ${props =>
    props.theme.sidebar.footer.background} !important;
  padding: ${props => props.theme.spacing(3)}px
    ${props => props.theme.spacing(4)}px;
  border-right: 1px solid rgba(0, 0, 0, 0.12);
`;

const SidebarFooterText = styled(Typography)`
  color: ${props => props.theme.sidebar.footer.color};
`;
const Dot = styled.span`
  width: 12px;
  height: 12px;
  margin-right: 4px;
  background-color: ${props => props.theme.sidebar.footer.online.background};
  display: inline-block;
  border-radius: 50%;
  margin-bottom: -0.5px;
`;

//
const CategoryText = styled(ListItemText)`
  margin: 0;
  span {
    color: ${props => props.theme.sidebar.color};
    font-size: ${props => props.theme.typography.body1.fontSize};
    padding: 0 16px;
  }
`;

const LinkText = styled(ListItemText)`
  color: ${props => props.theme.sidebar.color};
  span {
    font-size: ${props => props.theme.typography.body1.fontSize};
  }
  margin-top: 0;
  margin-bottom: 0;
`;

const Link = styled(ListItem)`
  padding-left: ${props => props.theme.spacing(12)}px;
  padding-top: ${props => props.theme.spacing(2)}px;
  padding-bottom: ${props => props.theme.spacing(2)}px;

  span {
    color: ${props => rgba(props.theme.sidebar.color, 0.7)};
  }

  &:hover span {
    color: ${props => rgba(props.theme.sidebar.color, 0.9)};
  }

  &.${props => props.activeClassName} {
    background-color: ${props => darken(0.06, props.theme.sidebar.background)};

    span {
      color: ${props => props.theme.sidebar.color};
    }
  }
`;

const Category = styled(ListItem)`
  padding: ${props => props.theme.spacing(3)}px;

  svg {
    color: ${props => props.theme.sidebar.color};
    font-size: 20px;
    width: 20px;
    height: 20px;
    opacity: 0.5;
  }

  &:hover {
    background: rgba(0, 0, 0, 0.08);
  }

  &.${props => props.activeClassName} {
    background-color: ${props => darken(0.05, props.theme.sidebar.background)};

    span {
      color: ${props => props.theme.sidebar.color};
    }
  }
`;

const CategoryIconLess = styled(ExpandLess)`
  color: ${props => rgba(props.theme.sidebar.color, 0.5)};
`;

const CategoryIconMore = styled(ExpandMore)`
  color: ${props => rgba(props.theme.sidebar.color, 0.5)};
`;

const Scrollbar = styled(PerfectScrollbar)`
  background-color: ${props => props.theme.sidebar.background};
  border-right: 1px solid rgba(0, 0, 0, 0.12);
`;

const List = styled(MuiList)`
  background-color: ${props => props.theme.sidebar.background};
`;

const Items = styled.div`
  padding-top: ${props => props.theme.spacing(2.5)}px;
  padding-bottom: ${props => props.theme.spacing(2.5)}px;
`;

export {
  Drawer,
  Brand,
  Dot,
  SidebarFooter,
  SidebarFooterText,
  CategoryText,
  LinkText,
  Link,
  Category,
  CategoryIconLess,
  CategoryIconMore,
  Scrollbar,
  List,
  Items,
};
