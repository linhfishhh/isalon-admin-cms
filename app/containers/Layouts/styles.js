import styled from 'styled-components';
import { spacing } from '@material-ui/system';
import { Paper as MuiPaper } from '@material-ui/core';
import Navbar from 'components/Navbar';

const drawerWidth = 250;

const Root = styled.div`
  display: flex;
  min-height: 100vh;
`;

const Drawer = styled.div`
  ${props => props.theme.breakpoints.up('lg')} {
    width: ${drawerWidth}px;
    flex-shrink: 0;
  }
`;

const AppContent = styled.div`
  flex: 1;
  display: flex;
  flex-direction: column;
`;

const Paper = styled(MuiPaper)(spacing);

const MainContent = styled(Paper)`
  flex: 1;
  background: ${props => props.theme.body.background};
  padding: ${props => props.theme.spacing(4)}px;

  .MuiPaper-root {
    border: 1px solid #e6eaee;

    .MuiPaper-root {
      border: 0;
    }
  }
`;

const NavBar = styled(Navbar)`
  z-index: ${props => props.theme.zIndex.drawer + 1};
`;

const MainSinglePage = styled.div`
  flex: 1;
`;

export {
  drawerWidth,
  Root,
  AppContent,
  MainContent,
  Drawer,
  NavBar,
  MainSinglePage,
};
