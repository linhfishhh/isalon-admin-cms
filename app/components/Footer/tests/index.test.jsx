/**
 * Testing the NotFoundPage
 */

import React from 'react';
import renderer from 'react-test-renderer';
import { ThemeProvider } from '@material-ui/styles';
import Theme from 'components/Themes';
import Footer from '../index';

describe('<Footer />', () => {
  it('should render the Footer', () => {
    const footer = renderer
      .create(
        <ThemeProvider theme={Theme}>
          <Footer />
        </ThemeProvider>,
      )
      .toJSON();
    expect(footer).toMatchSnapshot();
  });
});
