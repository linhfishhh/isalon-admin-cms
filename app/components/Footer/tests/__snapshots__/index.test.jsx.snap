// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render the Footer 1`] = `
<div
  className="makeStyles-root-1"
>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    ©
     
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="/"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_blank"
    >
      AnhNT
    </a>
    . 2019
  </p>
  <span
    className="MuiTypography-root MuiTypography-caption"
  >
    Created with love for the environment. By designers and developers who love to work together in offices!
  </span>
</div>
`;

exports[`<Footer /> should render the Page Not Found text 1`] = `
<div
  className="makeStyles-root-1"
>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    ©
     
    <a
      className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
      href="/"
      onBlur={[Function]}
      onFocus={[Function]}
      target="_blank"
    >
      AnhNT
    </a>
    . 2019
  </p>
  <span
    className="MuiTypography-root MuiTypography-caption"
  >
    Created with love for the environment. By designers and developers who love to work together in offices!
  </span>
</div>
`;
